+neurostim/design.m

```diff
@@ -95,13 +95,13 @@
         retry = 'IGNORE'; %IGNORE,IMMEDIATE,RANDOM
         weights=1;               % The relative weight of each condition. (Must be scalar or the same size as specs)
         maxRetry = Inf;
+        list;                    % The order in which conditions will be run.
```

Comment:
Maybe a better solution is to add a second input argument to shuffle() , which specifies the new list order. 
Shuffle can then make sure that the randomization member is set appropriately (SEQUENTIAL, probably, or does this need an additional mode like MANUAL?).
That avoids creating an object where the randomization member and the actual list are contradictory.